.github/ISSUE_TEMPLATE.md

```diff
@@ -36,6 +36,10 @@ or sensitive account information (API keys, credentials, etc.) when reporting an
 
 [What percentage of the time does it reproduce?]
 
+#### Twilio Call SID(s)
+
+Twilio Call SID when the issue happened. You can find Call SIDs in TVOCall.sid or TVOCallInvite.callSid. Call SIDs are also available in https://www.twilio.com/console/voice/calls/logs.
```

Comment:
> You can find the Call SID in the SDK from [TVOCall.sid](https://twilio.github.io/twilio-voice-ios/docs/latest/Classes/TVOCall.html#//api/name/sid) or [TVOCallInvite.callSid](https://twilio.github.io/twilio-voice-ios/docs/latest/Classes/TVOCallInvite.html#//api/name/callSid) or in the [Twilio Calls Console](https://www.twilio.com/console/voice/calls/logs).

Reply:
I wanted to use formatted docs/logs links but then I realized that when developers are opening issues they will only see the raw text instead of the rendered format.

Reply:
Ah okay, then please use this language and keep the console link as you had it before.